packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -94,20 +99,26 @@ export const DropdownContent = (
     {
         side = 'bottom',
         align = 'start',
+        distance = 'compact',
```

Comment:
We use `padding` everywhere else
```suggestion
        padding = 'compact',
```

Reply:
as we discussed I was confused by the naming, since this seems to be a very niche configuration that I expect will only be used once in a blue moon I wouldn't be opposed to giving it a braindead name like `triggerOffset` or `triggerGap`